uniffi_bindgen/src/bindings/kotlin/templates/Helpers.kt

```diff
@@ -3,19 +3,21 @@
 // Error runtime.
 @Structure.FieldOrder("code", "error_buf")
 internal open class RustCallStatus : Structure() {
-    @JvmField var code: Int = 0
+    @JvmField var code: Byte = 0
```

Comment:
I think this has been the wrong type since it was introduced :exploding_head:

Reply:
It doesn't matter much, but we define it as a `i8` in the `rustcalls.rs` file.  I think the main reason for `i8` was Kotlin's historical issues with unsigned types.  I don't think that's much of an issue anymore, at some point we could consider changing a lot of the FFI types to be unsigned.